addons/packages/app-toolkit/0.1.0/test/app-toolkit-test.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/creack/pty"
+)
+
+const workloadURL = "http://tanzu-simple-web-app.default.127-0-0-1.sslip.io/"
+
+func main() {
+	fmt.Println("STARTING TEST")
+
+	if checkCommand("tanzu uc list", "app-toolkit-test") {
+		runCommand("tanzu uc delete app-toolkit-test")
+	}
+
+	fmt.Println("\nTEST STEP Executable check")
+	validateCommand("tanzu", "Tanzu CLI")
+	validateCommand("tanzu apps", "Applications on Kubernetes")
+	validateCommand("tanzu package", "Tanzu package management")
+	validateCommand("kubectl", "kubectl controls the Kubernetes cluster manager")
+	validateCommand("docker", "A self-sufficient runtime for containers")
+	validateCommand("ytt version", "ytt version")
+	fmt.Println("TEST STEP Executable check OK")
+
+	fmt.Println("\nTEST STEP Install TCE and wait for ready")
+	runCommand("tanzu uc create app-toolkit-test -p 80:80 -p 443:443")
+	pollCommand("kubectl get nodes", " Ready", 10)
+	fmt.Println("TEST STEP Install TCE and wait for ready OK")
+
+	fmt.Println("\nTEST STEP Install dev package PRERELEASE ONLY")
+	runCommand("tanzu package repository update projects.registry.vmware.com-tce-main-v0.11.0 -n tanzu-package-repo-global --url projects.registry.vmware.com/tce/main:v0.11.0-alpha.3568.3")
+	fmt.Println("TEST STEP Install dev package PRERELEASE ONLY OK")
+
+	fmt.Println("\nTEST STEP Install app-toolkit")
```

Comment:
```suggestion
	fmt.Println("TEST STEP Install TCE and wait for ready OK")

	fmt.Println("\nTEST STEP Install app-toolkit")
```
Let's back out updating this package repo